src/components/ActivitiesListFilters.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {setTextFilter} from '../actions/filters';
+
+const ActivitiesListFilters = (props) => (
```

Comment:
destructure props. Ohh... I forgot -> learn about `prop-types` package and implement it (by separate PR). Easy to handle ;) use documentation :P